pact/http/consumer-two/build.gradle.kts

```diff
@@ -15,3 +16,15 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("au.com.dius.pact.consumer:junit5")
 }
+
+project.version = "1.5.0"
+
+pact {
+    publish {
+        pactBrokerUrl = "http://localhost:9292"
+        consumerVersion = project.version
+        pactBrokerUsername = "pact-user"
+        pactBrokerPassword = "123"
+    }
+}
```

Comment:
see comments / questions in `consumer-one`

Reply:
done